lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components",
-  "version": "7.4.4",
+  "version": "7.5.0",
```

Comment:
Why is this a minor bump and not a patch?

Reply:
> Why is this a minor bump and not a patch?

I needed some clarification on this as well. As most of my prior commits were bug fixes, I went with patches. Since this was more of an addition, I thought it would be more of a minor bump.

Reply:
> I needed some clarification on this as well. As most of my prior commits were bug fixes, I went with patches. Since this was more of an addition, I thought it would be more of a minor bump.

Keep in mind the version are primarily versions of the library, not the gallery.  As such any PRs which only change the gallery are automatically patches.